src/builder/create_interaction_response.rs

```diff
@@ -275,9 +275,32 @@ impl CreateInteractionResponseMessage {
     super::button_and_select_menu_convenience_methods!(self.components);
 }
 
-/// They're same according to Discord, see
-/// [Autocomplete docs](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-autocomplete).
-pub type AutocompleteChoice = CommandOptionChoice;
+// Same as CommandOptionChoice according to Discord, see
+// [Autocomplete docs](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-autocomplete).
+#[must_use]
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
```suggestion
#[must_use]
#[serde(transparent)]
#[derive(Clone, Debug, Deserialize, Serialize)]
```

To be safe.

Reply:
If it makes you feel better :p

Reply:
I guess for json it doesn't make a difference, see: https://serde.rs/json.html. Feel free to revert this if you feel like it. I doubt Discord is changing their API's data format anytime soon :P